website/src/documentation/sdks/python-type-safety.md

```diff
@@ -91,8 +92,8 @@ The following code declares an `int` input and a `str` output type hint on the `
 ```
 
 The following code declares `int` input and output type hints on `filter_evens`, using annotations on `FilterEvensDoFn.process`.
-Since `process` returns a generator, the output type is annotated as `Iterable[int]` (`Generator[int, None, None]` would also work here).
-Beam will remove the outer iterable of the return type on the `DoFn.process` method and functions passed to `ParDo` and `FlatMap`.
```

Comment:
If you're removing mentions of ParDo here, then the corresponding snippets should also be updated to use FlatMap instead of ParDo.

Reply:
The ParDos in the snippets are being passed DoFns. I removed the reference to passing (raw) functions to ParDo.